src/webgpu/web-platform/reftests/gpu_ref_test.ts

```diff
@@ -0,0 +1,49 @@
+import { DevicePool } from '../../../common/framework/gpu/implementation.js';
+import { assert, unreachable } from '../../../common/framework/util/util.js';
+
+const devicePool = new DevicePool();
```

Comment:
We don't need to use the DevicePool for this test. It will always run independently of other tests (not on the same page load), and so won't need to share Devices with anything else. You can just create a device the usual way.

Reply:
Actually don't even need to use getGPU as we can assume these are running in the browser (and not Node or something).